Mobile/CarWashService/SignUpView.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  SignUpView.swift
+//  CarWashService
+//
+//  Created by Anna Boykova on 18/11/2019.
+//  Copyright © 2019 VXDESIGN.STORE. All rights reserved.
+//
+
+import SwiftUI
+
+struct SignUpView: View {
+    @State private var firstName = ""
+    @State private var secondName = ""
+    @State private var emailAddress = ""
+    @State private var password = ""
+    @State private var confirmPassword = ""
+    
+    var body: some View {
+        VStack{
+            LogoView(subTitle: SignUpTitleStr).edgesIgnoringSafeArea(.top)
+            TextField("John", text: $firstName)
+                .textFieldStyle(RoundedBorderTextFieldStyle())
+                .padding(5)
+            TextField("Smith", text: $secondName)
+                .textFieldStyle(RoundedBorderTextFieldStyle())
+                .padding(5)
+            TextField("johnnyappleseed@apple.com", text: $emailAddress)
```

Comment:
I consider to set the following placeholders:

- First Name
- Last Name
- Email Address